tsconfig-react.json

```diff
@@ -0,0 +1,14 @@
+{
+	"compilerOptions": {
+		"module": "es2015",
+		"target": "es5",
+		"outDir": "dist",
+		"declarationDir": "dist",
+		"declaration": true,
+		"moduleResolution": "node",
+		"allowSyntheticDefaultImports": true,
+		"esModuleInterop": true,
+		"experimentalDecorators": true
+	},
+	"include": ["src/react/index.ts"]
+}
```

Comment:
Super minor nit, but could you rename this file as `tsconfig.react.json` to match the convention already established by `tsconfig.eslint.json`?

Reply:
resolved by 0929fbbc93f8732e020c755d04d07a132ee493ac